### PR TITLE
[not needed anymore] cmake: add `KOKKOS_STDBLAS_ENABLE_TESTS` cmake var to config file 

### PR DESCRIPTION
**Pull request by fnrizzi**

### CODE REVIEW

File: include/experimental/__p1673_bits/linalg_config.h.in [hunk 1]
```diff
@@ -4,3 +4,4 @@
 #cmakedefine LINALG_ENABLE_CONCEPTS
 #cmakedefine LINALG_ENABLE_KOKKOS
 #cmakedefine LINALG_ENABLE_KOKKOS_DEFAULT
+#cmakedefine LINALG_ENABLE_TESTS
```
Comment (dalg24) [inline include/experimental/__p1673_bits/linalg_config.h.in:7]: Why do you need it?

Comment (fnrizzi) [reply include/experimental/__p1673_bits/linalg_config.h.in:7]: I was using it inside the kokkos impl to conditionally make a print that is detected from the tests

**Comment by fnrizzi** [comment]
@dalg24 this is just temporary, see this https://github.com/kokkos/stdBLAS/issues/154

**Comment by fnrizzi** [comment]
please note that this variable will be moved out of that config file once we move the kokkos impl of the blas functions

**Comment by fnrizzi** [comment]
@dalg24 I renamed it for now, if you have preferences on the name let me know

**Pull request closed without merging**